examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx

```diff
@@ -64,12 +58,9 @@ export default function LivenessDefault({
                   const response = await handleGetLivenessDetection(
                     createLivenessSessionApiData.sessionId
                   );
-                  if (!response.isLive) {
-                    setCheckFailed(true);
-                  }
                 }}
                 onError={(error) => {
-                  setError(error);
+                  console.error(error);
```

Comment:
Are we not surfacing errors to end users?

Reply:
We have a default error modal that shows up, this is just another error callback for developers who want custom error handling